src/explore-education-statistics-common/src/components/ScrollableList.tsx

```diff
@@ -0,0 +1,16 @@
+import classNames from 'classnames';
+import React, { ReactNode } from 'react';
+import styles from './ScrollableList.module.scss';
+
+interface Props {
+  children: ReactNode;
+  className?: string;
+}
+
+const ScrollableList = ({ children, className = 'govuk-list' }: Props) => {
```

Comment:
Probably wouldn't go with a component like this as it's less flexible than having something like:

```tsx
export default function ScrollableContainer({ children, className }: Props) {
  return <div className={classNames(styles.container, className)}>{children}</div>;
} 
```

Which just gets used like:

```tsx
<Details>
  <ScrollableContainer>
    <ul>...</ul>
  </ScrollableContainer>
</Details>
```

Personally, a util class for adding scollable overflow, and a `style` prop would have sufficed, but the component approach is fine for now.